eng/intellisense.targets

```diff
@@ -25,13 +27,26 @@
     <PackageDownload Include="Microsoft.Private.Intellisense" Version="[$(MicrosoftPrivateIntellisenseVersion)]" />
   </ItemGroup>
 
+  <!-- Prepare the intellisense package xml file by copying it to the project's intermediate folder and update its file timestamp.
+       This is necessary so that all project outputs are newer than all project inputs. Directly copying from the intellisense package
+       would violate that and break fast up-to-date check. -->
+  <Target Name="PrepareIntellisensePackageXmlFile"
+          Inputs="$(IntellisensePackageXmlFilePath)"
+          Outputs="$(IntermediateDocFileItemFromIntellisensePackage)">
+    <Copy SourceFiles="$(IntellisensePackageXmlFilePath)"
+          DestinationFiles="$(IntermediateDocFileItemFromIntellisensePackage)" />
+
+    <Touch Files="$(IntermediateDocFileItemFromIntellisensePackage)" />
```

Comment:
```suggestion
    <Touch Files="$(IntermediateDocFileItemFromIntellisensePackage)" />
```

Reply:
These empty lines are intentional for readability.